std/experimental/allocator/package.d

```diff
@@ -226,6 +226,22 @@ module std.experimental.allocator;
 public import std.experimental.allocator.common,
     std.experimental.allocator.typed;
 
+shared static this()
```

Comment:
object.Error@src/rt/minfo.d(371): Cyclic dependency between module std.experimental.allocator and std.parallelism
std.experimental.allocator* ->
std.algorithm.comparison ->
std.parallelism* ->
std.file ->
std.experimental.checkedint ->
std.experimental.allocator.common ->
std.experimental.allocator*